core/src/main/java/com/scalar/db/storage/dynamo/SelectStatementHandler.java

```diff
@@ -1,24 +1,30 @@
 package com.scalar.db.storage.dynamo;
```

Comment:
I think the changes in `SelectStatementHandler` are most significant and a little bit tricky.